### ISSUE TITLE
EDIT: p-value

### ISSUE THREAD

**Issue by JulianeKloidt** [PsyTeachR/glossary#79]
I would suggest adding some caution statement linking the p-value to assumptions of the statistical model:

P-Value (= observed significance level) is a statistical summary of the compatibility between the observed data and what we would predict or expect to see if we knew the entire statistical model (all the assumptions used to compute the P value) were correct. The P value tests all the assumptions about how the data were generated, not just the targeted hypothesis it is supposed to test (e.g. null hypothesis). The P value assumes all assumptions (including the test hypothesis) are true. 
A p-value represents a conditional probability given that H0 is correct. P-values do not provide information about H1.

**Comment by JulianeKloidt** [comment]
yes, it's from my course notes from the Bootstrap module with Guillaume in 2020

**Comment by debruine** [comment]
I find this bit a little confusing. How are you mapping "targeted hypothesis", "null hypothesis" and "test hypothesis" onto H0 and H1?

> The P value tests all the assumptions about how the data were generated, not just the targeted hypothesis it is supposed to test (e.g. null hypothesis). The P value assumes all assumptions (including the test hypothesis) are true.
